actions/class.ItemContent.php

```diff
@@ -202,7 +202,7 @@ private function getMediaAsset(bool $validateWriteAccess = false): ?MediaAsset
     {
         $params = $this->getRequiredQueryParams('uri', 'lang', 'path');
         $asset = $this->resolveAsset($params['uri'], $params['path'], $params['lang']);
-        $resourceUri = tao_helpers_Uri::decode($asset->getMediaIdentifier());
+        $resourceUri = $params['uri'];
```

Comment:
We need to make sure it wont break ACL for assets (not exclusive item files). It should support both ways.

Reply:
It is working fine for assets too.

Reply:
And for assets, will the `$params['uri']` be the item URI or the asset URI?

Reply:
It is Item URI

Reply:
So, that means that we do not check permissions for the asset properly.